src/components/questionary/Questionary.tsx

```diff
@@ -34,6 +34,7 @@ const useStyles = makeStyles((theme) => ({
   },
   root: {
     width: 'inherit',
+    minWidth: '500px',
```

Comment:
This one once added and removed in (https://github.com/UserOfficeProject/user-office-frontend/blob/80d00917969c87bfaa3b2d3bf4f8971fae43209d/src/components/questionary/Questionary.tsx#L36), now it's back, I think we should talk about this part or leave a comment why is it needed.

Reply:
I think you mean L:37 minWidth?
That is if questions are short the view looks squashed. It is the case for shipment review where there are just few questions, but could be the case for other questionnaires as well.

Reply:
I meant why is it needed there, so others won't remove it in the future like they did previously.